docs/docs/icicle/primitives/msm.md

```diff
@@ -49,14 +49,18 @@ Accelerating MSM is crucial to a ZK protocol's performance due to the [large per
 
 You can learn more about how MSMs work from this [video](https://www.youtube.com/watch?v=Bl5mQA7UL2I) and from our resource list on [Ingopedia](https://www.ingonyama.com/ingopedia/msm).
 
-# Using MSM
-
 ## Supported curves
 
 MSM supports the following curves:
 
 `bls12-377`, `bls12-381`, `bn-254`, `bw6-761`, `grumpkin`
```

Comment:
```suggestion
`bls12-377`, `bls12-381`, `bn254`, `bw6-761`, `grumpkin`
```
I think its standard that bn254 doesn't have a `-` between the letters and the numbers unlike the other supported curves